renderer/components/ActivitiesTable.tsx

```diff
@@ -66,11 +66,24 @@ export default function ActivitiesTable({
         console.error("Clipboard write error:", error);
       });
   };
+  useEffect(() => {
+    const handleKeyDown = (event) => {
```

Comment:
Why you define the function inside useEffect? I think you can do it inside component body to prevent making new function every time when activity was changed